### PR TITLE
fix: proplerly render extraContainers

### LINKED ISSUE #16470: Helm chart incorrectly renders extraContainers
## Environment
- **Airbyte version**:  0.40.3
- **Chart Version**: 0.45.1
- **OS Version / Instance**: helm 3.9.4, macOS 12.3.1
- **Deployment**: kubernetes/helm
- **Source Connector and version**: (if applicable example Salesforce 0.2.3) 
- **Destination Connector and version**: (if applicable example Postgres 0.3.3) 
- **Step where error happened**: Deploy 

## Current Behavior
```
Error: YAML parse error on airbyte/charts/worker/templates/deployment.yaml: error converting YAML to JSON: yaml: line 300: mapping values are not allowed in this context
helm.go:84: [debug] error converting YAML to JSON: yaml: line 300: mapping values are not allowed in this context
YAML parse error on airbyte/charts/worker/templates/deployment.yaml
helm.sh/helm/v3/pkg/releaseutil.(*manifestFile).sort
	helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:146
helm.sh/helm/v3/pkg/releaseutil.SortManifests
	helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:106
helm.sh/helm/v3/pkg/action.(*Configuration).renderResources
	helm.sh/helm/v3/pkg/action/action.go:165
helm.sh/helm/v3/pkg/action.(*Install).RunWithContext
	helm.sh/helm/v3/pkg/action/install.go:259
main.runInstall
	helm.sh/helm/v3/cmd/helm/install.go:264
main.newTemplateCmd.func2
	helm.sh/helm/v3/cmd/helm/template.go:82
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.4.0/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.4.0/command.go:974
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.4.0/command.go:902
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:255
runtime.goexit
	runtime/asm_arm64.s:1133
```

### incorrectly rendered yaml
```yaml
        resources:
          limits: {}
          requests: {}
        volumeMounts:
        - name: gcs-log-creds-volume
          mountPath: /secrets/gcs-log-creds
          readOnly: true
              - args:
          - -ip_address_types=PRIVATE
          - -instances=random-id-123456:us-east1:airbyte=tcp:5432
          - -log_debug_stdout=true
          - -structured_logs
          command:
          - /cloud_sql_proxy
          env:
          - name: GOOGLE_APPLICATION_CREDENTIALS
            value: /secrets/key.json
          image: gcr.io/cloudsql-docker/gce-proxy:1.26.0
          name: cloud-sql-proxy
          securityContext:
            runAsNonRoot: true
          volumeMounts:
          - mountPath: /secrets/
            name: service-account
```

## Expected Behavior
`extraContainers` in values are properly rendered.

### sample correctly rendered yaml
```yaml
        resources:
          limits: {}
          requests: {}
        volumeMounts:
        - name: gcs-log-creds-volume
          mountPath: /secrets/gcs-log-creds
          readOnly: true

      - args:
        - -ip_address_types=PRIVATE
        - -instances=random-id-123456:us-east1:airbyte=tcp:5432
        - -log_debug_stdout=true
        - -structured_logs
        command:
        - /cloud_sql_proxy
        env:
        - name: GOOGLE_APPLICATION_CREDENTIALS
          value: /secrets/key.json
        image: gcr.io/cloudsql-docker/gce-proxy:1.26.0
        name: cloud-sql-proxy
        securityContext:
          runAsNonRoot: true
        volumeMounts:
        - mountPath: /secrets/
          name: service-account
```

## Steps to Reproduce
1. Add additional containers in `worker.extraContainers`, `global.extraContainers`, `metrics.extraContainers`, `server.extraContainers`, `temporal.extraContainers`, or `webapp.extraContainers`
2. Attempt to install or upgrade the helm chart

## Are you willing to submit a PR?
yes

### PR DESCRIPTION
**Pull request by bdashrad**
Signed-off-by: Brad Clark <bdashrad@gmail.com>

## What
Fixes #16470 

## How
Apply proper indentation when rendering extraContainers

## 🚨 User Impact 🚨
Bug fix, no breaking changes

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Code reviews completed
- [ ] Documentation updated
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Comment by bdashrad** [comment]
It's unclear to me where the charts are actually stored, because changes in this code repo don't line up with the helm chart repo.

I'm also happy to run the helm-docs commands to update the README if they are run in ci/cd.

**Comment by bdashrad** [comment]
I have run [chart-testingn](https://github.com/helm/chart-testing/) locally, and there are some additional changes it would like to see, around newlines in values.yaml, version pinning in https://github.com/airbytehq/airbyte/blob/master/charts/airbyte/Chart.yaml#L26.

The airbyte-metrics chart is also missing from the helm chart repo, which causes further errors.

<details><summary><strong>chart-testing</strong></summary>
```
$ ct lint charts/airbyte
Linting charts...

------------------------------------------------------------------------------------------------------------------------
 Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 metrics => (version: "0.45.2", path: "charts/airbyte-metrics")
 server => (version: "0.45.2", path: "charts/airbyte-server")
 temporal => (version: "0.45.2", path: "charts/airbyte-temporal")
 webapp => (version: "0.45.2", path: "charts/airbyte-webapp")
 worker => (version: "0.45.2", path: "charts/airbyte-worker")
 airbyte => (version: "0.45.2", path: "charts/airbyte")
------------------------------------------------------------------------------------------------------------------------

Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "runatlantis" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "privatebin" chart repository
...Successfully got an update from the "falcosecurity" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "doppler" chart repository
...Successfully got an update from the "bdashrad" chart repository
...Successfully got an update from the "skm" chart repository
...Successfully got an update from the "argo" chart repository
...Successfully got an update from the "datadog" chart repository
...Successfully got an update from the "airbyte-oss" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "redis" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading common from repo https://charts.bitnami.com/bitnami
Deleting outdated charts
Linting chart 'metrics => (version: "0.45.2", path: "charts/airbyte-metrics")'
Checking chart 'metrics => (version: "0.45.2", path: "charts/airbyte-metrics")' for a version bump...
Old chart version: 0.39.36
New chart version: 0.45.2
Chart version ok.
Validating /Users/bradclark/code/airbyte/charts/airbyte-metrics/Chart.yaml...
Validation success! 👍
charts/airbyte-metrics/values.yaml
  1:1       error    too many blank lines (1 > 0)  (empty-lines)

Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "bdashrad" chart repository
...Successfully got an update from the "privatebin" chart repository
...Successfully got an update from the "doppler" chart repository
...Successfully got an update from the "skm" chart repository
...Successfully got an update from the "falcosecurity" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "runatlantis" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "argo" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "airbyte-oss" chart repository
...Successfully got an update from the "datadog" chart repository
...Successfully got an update from the "stable" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "redis" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading common from repo https://charts.bitnami.com/bitnami
Deleting outdated charts
Linting chart 'server => (version: "0.45.2", path: "charts/airbyte-server")'
Checking chart 'server => (version: "0.45.2", path: "charts/airbyte-server")' for a version bump...
Old chart version: 0.39.36
New chart version: 0.45.2
Chart version ok.
Validating /Users/bradclark/code/airbyte/charts/airbyte-server/Chart.yaml...
Validation success! 👍
charts/airbyte-server/values.yaml
  1:1       error    too many blank lines (1 > 0)  (empty-lines)
  193:1     error    too many blank lines (1 > 0)  (empty-lines)

Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "doppler" chart repository
...Successfully got an update from the "privatebin" chart repository
...Successfully got an update from the "runatlantis" chart repository
...Successfully got an update from the "skm" chart repository
...Successfully got an update from the "falcosecurity" chart repository
...Successfully got an update from the "bdashrad" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "argo" chart repository
...Successfully got an update from the "airbyte-oss" chart repository
...Successfully got an update from the "datadog" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "stable" chart repository
...Successfully got an update from the "redis" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading common from repo https://charts.bitnami.com/bitnami
Deleting outdated charts
Linting chart 'temporal => (version: "0.45.2", path: "charts/airbyte-temporal")'
Checking chart 'temporal => (version: "0.45.2", path: "charts/airbyte-temporal")' for a version bump...
Old chart version: 0.39.36
New chart version: 0.45.2
Chart version ok.
Validating /Users/bradclark/code/airbyte/charts/airbyte-temporal/Chart.yaml...
Validation success! 👍
charts/airbyte-temporal/values.yaml
  1:1       error    too many blank lines (1 > 0)  (empty-lines)
  155:20    error    no new line character at the end of file  (new-line-at-end-of-file)

Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "bdashrad" chart repository
...Successfully got an update from the "runatlantis" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "privatebin" chart repository
...Successfully got an update from the "skm" chart repository
...Successfully got an update from the "doppler" chart repository
...Successfully got an update from the "falcosecurity" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "datadog" chart repository
...Successfully got an update from the "argo" chart repository
...Successfully got an update from the "airbyte-oss" chart repository
...Successfully got an update from the "redis" chart repository
...Successfully got an update from the "stable" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading common from repo https://charts.bitnami.com/bitnami
Deleting outdated charts
Linting chart 'webapp => (version: "0.45.2", path: "charts/airbyte-webapp")'
Checking chart 'webapp => (version: "0.45.2", path: "charts/airbyte-webapp")' for a version bump...
Old chart version: 0.39.36
New chart version: 0.45.2
Chart version ok.
Validating /Users/bradclark/code/airbyte/charts/airbyte-webapp/Chart.yaml...
Validation success! 👍
charts/airbyte-webapp/values.yaml
  177:1     error    too many blank lines (1 > 0)  (empty-lines)

Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "doppler" chart repository
...Successfully got an update from the "bdashrad" chart repository
...Successfully got an update from the "privatebin" chart repository
...Successfully got an update from the "runatlantis" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "falcosecurity" chart repository
...Successfully got an update from the "skm" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "datadog" chart repository
...Successfully got an update from the "airbyte-oss" chart repository
...Successfully got an update from the "argo" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "redis" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading common from repo https://charts.bitnami.com/bitnami
Deleting outdated charts
Linting chart 'worker => (version: "0.45.2", path: "charts/airbyte-worker")'
Checking chart 'worker => (version: "0.45.2", path: "charts/airbyte-worker")' for a version bump...
Old chart version: 0.39.36
New chart version: 0.45.2
Chart version ok.
Validating /Users/bradclark/code/airbyte/charts/airbyte-worker/Chart.yaml...
Validation success! 👍
Validating maintainers...
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "doppler" chart repository
...Successfully got an update from the "runatlantis" chart repository
...Successfully got an update from the "bdashrad" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "privatebin" chart repository
...Successfully got an update from the "falcosecurity" chart repository
...Successfully got an update from the "skm" chart repository
...Successfully got an update from the "datadog" chart repository
...Successfully got an update from the "argo" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "airbyte-oss" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "redis" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: dependency "airbyte-bootloader" has an invalid version/constraint format: improper constraint: placeholder
------------------------------------------------------------------------------------------------------------------------
No chart changes detected.
------------------------------------------------------------------------------------------------------------------------
Error: Error linting charts: Error building dependencies for chart 'airbyte => (version: "0.45.2", path: "charts/airbyte")': Error waiting for process: exit status 1
Error linting charts: Error building dependencies for chart 'airbyte => (version: "0.45.2", path: "charts/airbyte")': Error waiting for process: exit status 1
```
</details>

**Comment by xpuska513** [comment]
> It's unclear to me where the charts are actually stored, because changes in this code repo don't line up with the helm chart repo.
> 
> I'm also happy to run the helm-docs commands to update the README if they are run in ci/cd.

Charts from this repo are being packaged and then store in https://github.com/airbytehq/helm-charts repo
See [this](https://github.com/airbytehq/airbyte/actions/workflows/publish-helm-charts.yml) workflow for more details
For now the version mismatch was seen due to the action used in the workflow for generating semantic version be unstable from time to time(I'm looking for the root cause, but probably it's due to one PR was merged with non-semantic comit thus breaking the logic of action used + hardcoded values for version bump(commit flags))

**Comment by xpuska513** [comment]
> I have run [chart-testing](https://github.com/helm/chart-testing/) locally, and there are some additional changes it would like to see, around newlines in values.yaml, version pinning in https://github.com/airbytehq/airbyte/blob/master/charts/airbyte/Chart.yaml#L26.

I'll add chart-testing utility into a check workflow that will make sure everything's okay before approving PR(commit and PR have semantic format + cart-testing validation is successfull)

Thanks for pointing out!

**Comment by xpuska513** [comment]
@bdashrad, everything LGTM, but before approving I'll clone your repo and deploy airbyte locally.

**Comment by bdashrad** [comment]
Thanks @xpuska513, it looks like the chart version numbering has changed, so let me know what you'd like me to do there.

File: charts/airbyte-bootloader/values.yaml [hunk 1]
```diff
@@ -49,16 +58,15 @@ tolerations: []
...
-  requests: {}
   ## Example:
   ## limits:
   ##    cpu: 200m
   ##    memory: 1Gi
   limits: {}
+  ## requests:
+  ##    memory: 256Mi
+  ##    cpu: 250m
+  requests: {}
```
Comment (bdashrad) [inline charts/airbyte-bootloader/values.yaml:69]: This aligns this section to be the same as the other charts in the repo.

**Review by xpuska513 — APPROVED** [review]
LGTM
Thanks for your contribution!

**Pull request merged**